site/examples/data/4/App.svelte

```diff
@@ -0,0 +1,153 @@
+<script>
+  import * as Pancake from "@sveltejs/pancake";
+  import { fly, scale } from "svelte/transition";
+  import { quintOut } from "svelte/easing";
+  import { crossfade } from "svelte/transition";
+  import { flip } from "svelte/animate";
+  import data from "./data.js";
+
+  const [send, receive] = crossfade({
+    duration: d => Math.sqrt(d * 200),
+
+    fallback(node, params) {
+      const style = getComputedStyle(node);
+      const transform = style.transform === "none" ? "" : style.transform;
+
+      return {
+        duration: 600,
+        easing: quintOut,
+        css: t => `
+					transform: ${transform} scale(${t});
```

Comment:
It looks like you have a mix of tabs and spaces in this file. All the other examples use exclusively tabs